daphne_worker/src/storage_proxy.rs

```diff
@@ -82,12 +83,32 @@ use worker::{js_sys::Uint8Array, Delay, Env, Request, RequestInit, Response};
 
 const KV_BINDING_DAP_CONFIG: &str = "DAP_CONFIG";
 
+pub fn authorize(req: &Request, env: &Env) -> Option<worker::Result<Response>> {
```

Comment:
I'd suggest matching the pattern we folllow in the daphne crate (unauthorized_reason -> the reason why we can't let the request go forward.)